packages/cra-template/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@fluentui/cra-template",
-  "version": "0.2.1",
+  "version": "8.0.0",
   "description": "Create React App template for Fluent UI React",
   "repository": {
     "type": "git",
     "url": "https://github.com/microsoft/fluentui",
     "directory": "packages/cra-template"
   },
   "scripts": {
+    "lint": "eslint --ext .js,.ts,.tsx .",
```

Comment:
AFAIR ext is not needed since eslint v7

Reply:
Per the docs, by default it only lints `*.js` and anything explicitly matching `overrides`. So at least in my opinion it makes sense to include the extensions to ensure all the desired files are linted.
https://eslint.org/docs/user-guide/command-line-interface#-ext

Reply:
right, I'd consider to use more common syntax maybe -> glob

```
eslint ./**/*.{ts,tsx,js}
```

although that might not work within cmd.exe env ☹️

Reply:
I think to ensure cross-platform compatibility I'll leave it as-is.